src/vs/workbench/browser/parts/titlebar/windowTitle.ts

```diff
@@ -128,6 +131,13 @@ export class WindowTitle extends Disposable {
 				this.titleUpdater.schedule();
 			}
 		}));
+		this._register(this.accessibilityService.onDidChangeScreenReaderOptimized(() => {
+			if (this.accessibilityService.isScreenReaderOptimized() && !this.titleIncludesEditorState) {
+				this.titleUpdater.schedule();
+			} else if (!this.accessibilityService.isScreenReaderOptimized() && this.titleIncludesEditorState) {
+				this.titleUpdater.schedule();
+			}
+		}));
```

Comment:
I tend to put the if-condition together and then have the actual code that runs just one and not repeated if its identical.

Reply:
not sure I follow 🤔

Reply:
oh, you mean combine the statements, will do